scrips/calendarVars.js

```diff
@@ -2,5 +2,18 @@ const weekDaysNumber = 7;
 const hoursNumber = 24;
 const cellHeight = 49;
 const currentEventTileId = "current-event-tile";
+const defaultEventLength = 0.75; //hours
```

Comment:
By default time is measure in ms but if you use default measurement unit I would suggest to specify it in the variable name.

```suggestion
const defaultEventLengthInHours = 0.75; //hours
```

P.S in your case I think it should be minutes instead of hours. The reason why is that shortest event duration in theory can be 1 minute.